app/models/user.rb

```diff
@@ -140,7 +140,9 @@ def pending_profile
       pending = profiles.where(deactivation_reason: :in_person_verification_pending).or(
         profiles.where.not(gpo_verification_pending_at: nil),
       ).or(
-        profiles.where.not(fraud_pending_reason: nil),
+        profiles.where.not(fraud_review_pending_at: nil),
+      ).or(
+        profiles.where.not(fraud_rejection_at: nil),
```

Comment:
Are we missing a spec in user_spec that describes `pending_profile`'s behavior for fraud pending? I was surprised there weren't any spec changes for this method.

Reply:
There are tests but they still pass with this change. This is actually returning things back to the way they were previously. If a profile has a `fraud_pending_reason` it also has a `fraud_review_pending_at` and `fraud_rejection_at`. The tests use the factories so all of that footwork is handled there.